client/src/components/menuCategoryButtons.jsx

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
 
 const MenuCategoryButtons = props => (
-  props.restaurants.map(category => (
-    <button className="button-menu-header" type="button">{ category }</button>
-  ))
+  <div>
+    { props.restaurants.map(category => (
+      category.submenu_name === props.currentSubMenu
+        ? <button className="button-menu-link-active__3KrRGgFS button-menu-link__1j5RFIKk secondary__37FrTgAl base__1GdpdUto" type="button" id={category.submenu_name} onClick={props.handleMenuCategoryClick}>{category.submenu_name}</button>
```

Comment:
For styling, it's better to put them in separate line
```suggestion
        ? <button 
            className="button-menu-link-active__3KrRGgFS button-menu-link__1j5RFIKk secondary__37FrTgAl base__1GdpdUto"
            type="button" id={category.submenu_name} 
            onClick={handleMenuCategoryClick}
         >
           {category.submenu_name}
         </button>
```

Reply:
I will apply it!